src/components/algolia/algolia-course-related.tsx

```diff
@@ -0,0 +1,84 @@
+"use client"
+
+import {useMemo} from "react"
+import algoliasearch from "algoliasearch/lite"
+import {RelatedProducts} from "react-instantsearch"
+import {InstantSearchNext} from "react-instantsearch-nextjs"
+import ImageCard from "@components/patterns/image-card"
+import {H3} from "@components/elements/headers"
+import ActionLink from "@components/elements/action-link"
+
+type Props = {
+  objectID: string
+  appId: string
+  searchIndex: string
+  searchApiKey: string
+}
+
+type RelatedCourseItemProps = {
+  sum_course_catalog_number?: string
+  sum_course_class_number?: string
+  title?: string
+  photo?: string
+  url?: string
+}
+
+const RelatedCourseItem = ({item}: {item: RelatedCourseItemProps}) => {
```

Comment:
We can type the `item` using the type from the `src/components/algolia/results/summer-course/summer-course.tsx` file. Just export the type in that file and you can reuse it here.